ga-ccd-definition/ComplexTypes/GAJudgesHearingListGAspec.json

```diff
@@ -0,0 +1,193 @@
+[
+  {
+    "ID": "GAJudgesHearingListGAspec",
+    "ListElementCode": "listForHearingLabel",
+    "ElementLabel": "### How do you want to list for a hearing?",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "GAJudgesHearingListGAspec",
+    "ListElementCode": "hearingPreferencesPreferredType",
+    "ElementLabel": " ",
+    "HintText": " ",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "GAJudicialHearingListOptionsGAspec",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "GAJudgesHearingListGAspec",
+    "ListElementCode": "hearingPreferredLocationLabel",
+    "ElementLabel": "### Select a location",
+    "FieldType": "Label",
+    "FieldShowCondition": "hearingPreferencesPreferredType = \"In person\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "GAJudgesHearingListGAspec",
+    "ListElementCode": "hearingPreferredLocation",
+    "ElementLabel": " ",
```

Comment:
This and few other elements in this complex type do not have corresponding entries in the corresponding PoJo. These need to have in sync else in future when we deal with callbacks and add data to the response we will miss these and eventually end up wiping that data out.

Reply:
@smitatalawadekar-hmcts I will take a look now. cheers